integration-test/common/src/test/java/org/apache/uniffle/test/ShuffleServerFaultToleranceTest.java

```diff
@@ -244,7 +244,7 @@ public static MockedShuffleServer createServer(int id, File tmpDir) throws Excep
     String basePath = dataDir1.getAbsolutePath() + "," + dataDir2.getAbsolutePath();
     shuffleServerConf.set(ShuffleServerConf.RSS_STORAGE_TYPE, StorageType.MEMORY_LOCALFILE_HDFS.name());
     shuffleServerConf.setLong(ShuffleServerConf.FLUSH_COLD_STORAGE_THRESHOLD_SIZE, 450L);
-    shuffleServerConf.setInteger("rss.rpc.server.port", SHUFFLE_SERVER_PORT + id);
+    shuffleServerConf.setInteger("rss.rpc.server.port", SHUFFLE_SERVER_PORT + 20 + id);
```

Comment:
Why `+ 20` fix the problem? Can we set it to zero for any available port?

Reply:
There is no `bind error`. But it occurs that server is unavailable.

Reply:
> There is no `bind error`. But it occurs that server is unavailable.

So why is it related to the failure, is `shutdownNow` enough to fix the problem?

Reply:
No, `shutdownNow` just make the log more clearer, otherwise thread which haven't shutdown will log in  other test cases.

Reply:
OK, I'm testing it by repeating CI